py3status/py3.py

```diff
@@ -1037,6 +1037,15 @@ def play_sound(self, sound_file):
             c = shlex.split('{} {}'.format(cmd, sound_file))
             self._audio = Popen(c)
 
+    def get_composite_string(self, format_string):
+        """
+        Return a string from a Composite.
+        """
+        if isinstance(format_string, Composite):
+            return format_string.text()
+        else:
+            return ''
```

Comment:
Let's kill a line before we tuck this away.
```diff
diff --git a/py3status/py3.py b/py3status/py3.py
index f4ce677d..1a0582cc 100644
--- a/py3status/py3.py
+++ b/py3status/py3.py
@@ -1041,10 +1041,9 @@ class Py3:
         """
         Return a string from a Composite.
         """
-        if isinstance(format_string, Composite):
-            return format_string.text()
-        else:
+        if not isinstance(format_string, Composite):
             return ''
+        return format_string.text()
 
     def stop_sound(self):
         """
```
We also could just kill the whole thing and you use `text()`. instead, eg
```diff
+            composite = self.py3.safe_format(self.format_notification, wwan_data)
-            notification = self.py3.get_composite_string(composite)
+            notification = composite.text()
```

Reply:
what do you prefer for this?

Reply:
don't remember all story but in memory function was usefull for weird issue